docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -81,9 +81,8 @@ Once you have the prerequisites installed, you need to prepare your development
 
 2. (Optional but recommended) [Set up an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
 
-3. Install a code editor of your choice.
 
-   We highly recommend using [Visual Studio Code](https://code.visualstudio.com/) or [Atom](https://atom.io/). These are great, free and open source code editors.
+3. Install a code editor of your choice, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```

Comment:
```suggestion
3. Install a code editor of your choice. Nevertheless, we recommend [Visual Studio Code](https://code.visualstudio.com/), it's free and open source.
```
How about this?

Reply:
`Nevertheless` is just added fluff here, IMHO.

Reply:
We initially say 'You can install the code editor of your choice', but then we go very (IMO) cut and dry with our suggestion. Not a big deal of course - we can leave it as how Noami suggested. 

Maybe there is a less 'fluffy' word we can use to transition to our recommendation?

Reply:
```suggestion
3. Install a code editor of your choice. If you aren't sure which one to use, we recommend [Visual Studio Code](https://code.visualstudio.com/) — it's free and open source.
```

If we want a transition...